README.md

```diff
@@ -1 +1,115 @@
-# Capston-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    <!-- - [Tech Stack](#tech-stack) -->
+    <!-- - [Key Features](#key-features) -->
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+    <!-- - [Usage](#usage) -->
+    <!-- - [Run tests](#run-tests) -->
+    <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+<!-- - [🔭 Future Features](#future-features) -->
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  <!-- - [🙏 Acknowledgements](#acknowledgements) -->
+  <!-- - [❓ FAQ](#faq) -->
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [IT Live Anuual conference] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[IT Live Anuual conference]** is a website about IT Live Conference
+
+[Video Presentation](https://www.loom.com/share/9160b9208c9f41379bbf3dfe396ff78e)
```

Comment:
- [ ] Good job on making the improvements! However, as suggested by the previous reviewer, please record another video presentation and make sure to talk about **THE PIECE OF CODE that you are proud of** because currently, you have not mentioned anything concerning the codes 😃 this showcase your ability to explain codes and other technical concepts